src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -96,6 +97,16 @@ export const TradeConfirm = ({ history }: RouterProps) => {
       .toNumber(),
   )
 
+  const gasFeeTradeAmountPercentage = bnOrZero(fees?.fee)
```

Comment:
Can we rename/comment this? From looking at it, it is not very clear what this refers to, unless we look at the actual implementation.